parsec/stubs/pypac/__init__.pyi

```diff
@@ -1,19 +1,18 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
 from __future__ import annotations
 
-from typing import Optional
-from pypac.api import PACFile
+from typing import Optional, Any
 
-import requests
+# import requests
 
 def get_pac(
     url: Optional[str] = None,
     js: Optional[str] = None,
     from_dns: bool = True,
     timeout: int = 2,
     allowed_content_types: Optional[set[str]] = None,
-    sessions: Optional[requests.Session] = None,
+    # sessions: Optional[requests.Session] = None,
```

Comment:
If the param is not needed in the typing, we should remove it :thinking:

Reply:
Good catch,  I added this comment for testing purposes and forgot to remove it :+1: